vissl/data/README.md

```diff
@@ -126,6 +126,57 @@ stl10_binary/
     unlabeled_X.bin
 ```
 
+## Expected dataset structure for the other benchmark datasets
+
+VISSL supports benchmarks inspired by the [VTAB](https://arxiv.org/pdf/1910.04867.pdf) and [CLIP](https://cdn.openai.com/papers/Learning_Transferable_Visual_Models_From_Natural_Language_Supervision.pdf) papers, for which the datasets either:
+
+- Do not directly exist but are transformations of existing dataset
+- Are not in a format directly compatible with the `disk_folder` format of VISSL
+
+To run these benchmarks, the following data preparation scripts are mandatory:
+
+- `create_clevr_count_data_files.py`: to create a dataset from [CLEVR](https://cs.stanford.edu/people/jcjohns/clevr/) where the goal is to count the number of object in the scene
+- `create_clevr_dist_data_files.py`: to create a dataset from [CLEVR](https://cs.stanford.edu/people/jcjohns/clevr/) where the goal is to estimate the distance of the closest object in the scene
+- `create_euro_sat_data_files.py`: to transform the [EUROSAT](https://github.com/phelber/eurosat) dataset to the `disk_folder` format
+- `create_food101_data_files.py`: to transform the [FOOD101](https://data.vision.ee.ethz.ch/cvl/datasets_extra/food-101) dataset to the `disk_folder` format
+- `create_patch_camelyon_data_files.py`: to transform the [PatchCamelyon](https://github.com/basveeling/pcam) dataset to the `disk_folder` format
+- `create_small_norb_azimuth_data_files.py` to create a dataset from [Small NORB](https://cs.nyu.edu/~ylclab/data/norb-v1.0-small/) where the goal is to find the azimuth or the photographed object
```

Comment:
we should mention what the format created will be: `disk_filelist` or `disk_folder` :)

Reply:
Indeed :) It should be fixed at f618c50.